build.sh

```diff
@@ -6,12 +6,14 @@ RPM_RELEASE="0.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
 
 ROLE_NAME="oVirt.disaster-recovery"
 PACKAGE_NAME="ovirt-ansible-disaster-recovery"
+LOG_NAME="ovirt-dr"
```

Comment:
If you are going to put logs into /var/log/ovirt-dr then please provide logrotate setup similarly as we do in engine:

https://github.com/oVirt/ovirt-engine/blob/master/packaging/sys-etc/logrotate.d/ovirt-engine.in

Reply:
You also need to add dependency to logrotate, replace relative paths inside it (please take a look at GENERATED list and its usage in engine Makefile [1]) and install it into /etc/logrotate.d [2].

[1] https://github.com/oVirt/ovirt-engine/blob/master/Makefile#L211
[2] https://github.com/oVirt/ovirt-engine/blob/master/ovirt-engine.spec.in#L990